src/promptflow/promptflow/_core/metric_logger.py

```diff
@@ -45,6 +45,15 @@ def remove_metric_logger(self, logger_func: Callable):
 
 
 def log_metric(key, value, variant_id=None):
+    """Log a metric for current promptflow run.
```

Comment:
Do we have a CI to test which function has missing doc stings or we'll check them manually?